src/components/Web3Status/index.tsx

```diff
@@ -124,7 +130,7 @@ function Web3StatusInner() {
       return (
         <div
           id="web3-status-connected"
-          className="flex items-center px-3 py-2 text-bold rounded-lg bg-transparent text-sm"
+          className="flex items-center px-3 py-2 text-bold rounded-lg text-sm bg-dark-800 shadow-sm hover:bg-dark-700"
```

Comment:
look at this custom styling which makes it more similar to the neighbouring BCH and Ember flairs.